### DISCUSSION TITLE
Turnkey embed account email authentication 400 status

### DISCUSSION THREAD

**Discussion by nsnull0** [alchemyplatform/aa-sdk#899] [Issues] [answered]
Hello team!
I am encountering some issues that I need assistance with. Previously, our authentication process was functioning correctly, but as of yesterday, I have encountered a 400 error when attempting to authenticate via email.

Additionally, I am unsure if the attached screenshot is relevant, but currently, email authentication is not operational, and only passkey authentication seems to be working. Moreover, attempts to export the wallet or any method that access the turnkey API by alchemy signer are failing.

Could someone please investigate these issues at your earliest convenience? Your prompt attention to this matter would be greatly appreciated.

![Screenshot 2024-08-05 at 1 00 12](https://github.com/user-attachments/assets/a47156a5-93b2-4f2e-a690-3307a9b18180)

**Comment by avarobinson** [comment]
Hey @nsnull0 - sorry you're running in to issues!

Can you share some more details:
- What version of the SDK are you using?
- What SDK method(s) are erroring? Can you share a raw snippet of your implementation where this is failing?

**Comment by nsnull0** [comment, marked as answer]
3.18.2
```
await alchemySigner.authenticate({
                    email: state.email,
                    type: "email"
                })
```

I guess this error was not from alchemySDK, probably turnkey server or any backend.

for some reason now back to 200 status code without doing anything.. maybe fixed?

**Reply by avarobinson** [reply]
@nsnull0 Have you seen this error reoccur?

**Reply by nsnull0** [reply]
so far is good, no reoccur...
